androminion/res/values-de/strings-ui.xml

```diff
@@ -200,6 +214,10 @@
 	<string name="journey_token_initial">\u0020\u0020R\u0020\u0020</string>
 	<string name="token_plus_one_action_initial">A</string>
 	<string name="token_plus_one_buy_initial">K</string>
+	<string name="deluded_initial">\u0020\u0020D\u0020\u0020</string>
```

Comment:
These are supposed to be letters to represent the the states players can take. (e.g. D for Deluded, E for Envious, L for Lost in the Woods). If you want to translate them I would assume that they would be something like G for Getäuscht, N for Neidisch, and something else for "Im Wald verlaufen". There are 2 spaces on either side for spacing out the token to look better on the player status area.